isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/nutzungsvorgaben/inhalt.adoc

```diff
@@ -39,12 +89,68 @@ Hierzu sind zunächst die folgenden Abhängigkeiten erforderlich:
+|===
+
+[[isyfact-specific-client-registration]]
+==== Isyfact spezifische Konfigurationsparameter
+
+:linkaktuell: xref:client-credential-flow[]
+include::glossary:miscellaneous:partial$deprecated-abschnitt.adoc[]
+
+****
+*Anmerkung zur Deprecation*
+
+Dieser Flow soll gemäß OAuth2.0 Vorgaben nicht mehr verwendet werden und wird in zukünftigen Versionen aus dem OAuth-Standard entfernt.
+
+Die folgenden Konfigurationsparameter werden als Übergangslösung für technische Nutzer bereitgestellt.
+Der Client-Credentials Flow ist bevorzugt zu nutzen.
+****
+
+Zugangsdaten für technische Nutzer zur Authentifizierung von Tasks und Batches mit dem _Resource-Owner-Password-Credential_ Flow können über die folgenden Parameter konfiguriert werden:
+
+[[table-isyfact-client-registratoin-extension]]
+.Isyfact spezifische Erweiterung der Spring ClientRegistration
+[cols="3m,2m,2m,8",options="header"]
+|===
+|Parameter |Wertebereich |Default |Beschreibung
+|isy.security.oauth2.client.registration._<registrationId>_.username |String | null | Benutzername für Grant-Type `password`.
+|isy.security.oauth2.client.registration._<registrationId>_.password |String | null | Benutzerpassword für Grant-Type `password`.
+|isy.security.oauth2.client.registration._<registrationId>_.bhknz |String | null | Behördenkennzeichen für Grant-Type `password`. (optional)
+|isy.security.oauth2.client.bhknz-header-name |String | x-client-cert-bhknz |Konfigurierbarer Headername für Zwei-Faktor-Authentifizierung.
+|isy.security.oauth2.client.default-certificate-ou |String | null | Organisational Unit, die in Verbindung mit bhknz verwendet wird, um den Wert für bhknz-header-name zu bilden.
+
+(Muss gesetzt werden, wenn eine `registration` mit `bhknz` konfiguriert ist)
+|===
+
+[[authentifizierung_ccf]]
+=== Authentifizierung eines Clients mit Client Credentials
+
+Mit dem `OAuth 2.0`-Grant `client_credentials` steht eine vereinfachte Zugriffskontrolle für eine Client-Anwendung im Rahmen des `Client Credential Flow` zur Verfügung.
+
+Spring Security bietet umfassenden Support für `OAuth 2.0` und unterstützt die Authentifizierung interner Systeme auf Basis des `Client Credentials Flow`, wobei auf Spring interne Mechanismen zugegriffen wird.
 
-Um clientseitig die automatische Konfiguration einer Anwendung zu ermöglichen, werden zunächst die oben aufgeführten Abhängigkeiten eingebunden.
-Um im Rahmen des Client Credential Flows HTTP-Anfragen an einen Ressourcenserver stellen zu können, ist zusätzlich die Konfiguration des reaktiven `WebClient` erforderlich.
-Hierzu wird die Standardimplementierung mit dem Zusatz eines OAuth2-Autorisierungsfilters verwendet:
+Um clientseitig die automatische Konfiguration einer Anwendung zu ermöglichen, werden zunächst die oben aufgeführten Abhängigkeiten eingebunden und eine Client-Registration für den `Client Credentials Flow` angelegt.
```

Comment:
What's the intension of this sentence?
- is there realy an automatic configuration? In the second part of the sentence the manually created _Client-Registration configuration_ is mentioned
- Does it want to express, that with the correct configured dependencies and previous created _Client-Registration-Configuration_ a _confidential client application_ could be authenticated very easy by using simply the `oauth2ClientRegistrationId`?